piqe_ocp_lib/api/resources/ocp_secrets.py

```diff
@@ -61,6 +57,48 @@ def get_secret_token(self, secret_name, namespace="default"):
 
         return secret_token
 
+    def get_secret_names(self, namespace="default"):
+        """
+        Get service account secret token from openshift cluster
+        :param namespace: (string) name of namespace where secret exist
+        :return: (string) secret token for specified secret
+        """
+        api_response = None
+        secret_name_list = list()
+        try:
+            api_response = self.ocp_secret.get(namespace=namespace)
+        except ApiException as e:
```

Comment:
Moving forward we should raise the exception so the calling method can make a determination on whether to act or not.

Reply:
Sure